readme.md

```diff
@@ -38,64 +38,63 @@ $ npm -v
 
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
-* **Your Answer:**
+* **Your Answer:** When you Fork the repo then you are adding it under your repository.  Just cloning would two versions of the same repositoty.  If there
```

Comment:
Looks like this is unfinished. You're right on forking! Can't tell yet on cloning.  :)